rust/android/Cargo.toml

```diff
@@ -15,7 +18,8 @@ tokio =  { version = "1.24.2", features = ["full"] }
 zingo-testutils = { git="https://github.com/zingolabs/zingolib", default-features=true, branch = "dev" }
 json = "0.12.4"
 env_logger = "0.10.0"
-test_utils = { path = "../test_utils" }
+zingomobile_utils = { path = "../zingomobile_utils" }
+regchest_utils = { git="https://github.com/Oscar-Pepper/zingo-regchest", default-features=true, branch = "launch_regtest" }
```

Comment:
I need to understand all of this... Why you need to refer to this repo? I need to check it out...

Reply:
my plan was to work on my own repo and send a PR to the zingolabs repo for review... i actually messed that up by accidentally merging to zingolabs haha. its pointing at zingolabs now i have *hopefully* finished